src/js/service/ImportService.js

```diff
@@ -130,4 +130,75 @@
       return pskl.utils.FrameUtils.createFromImage(resizedImage);
     });
   };
+
+  /**
+   * @param {pskl.model.Piskel} originalPiskel
+   * @param {pskl.model.Piskel} additionalPiskel
+   * @returns {pskl.model.Piskel}
+   */
+  // Adds frames from an additionalPiskel to an originalPiskel, adjusting the size of
+  // both piskels to the largest demensions of either piskel, and centering the content.
+  // This will only work when there is only one layer.
+  // Multi layer animations are disabled from the UI in code-dot-org/piskel
+  ns.ImportService.prototype.addTwoPiskels = function(originalPiskel, additionalPiskel) {
```

Comment:
I wonder if there's a better name for this. `addTwoPiskels` makes me think we are generating a third piskel from two existing piskels, when really what's happening is we're glomming one piskel onto another, potentially mutating either of them along the way. Maybe `appendPiskel` or `mergePiskels`?